Notify/Notify/Notify/AppShell.xaml.cs

```diff
@@ -35,6 +43,35 @@ public AppShell()
             }
         }
 
+        private void ConnetivityChangedHandeler(object sender, ConnectivityChangedEventArgs e)  // TODO: move into Notify.Android
+        {
+            var connectivityManager = (ConnectivityManager)Android.App.Application.Context.GetSystemService(Context.ConnectivityService);
```

Comment:
Consider using the actual type instead of `var` in all places, so that will be more readable.

Reply:
Done.
I replaced all the places where I was using 'var' with the right type.